### ISSUE TITLE
[BUG] JQ Filter doesn't catch on logs coming from LR Webhook Beat

### ISSUE THREAD

**Issue by TonyMasse** [logrhythm/EZ-Cloud#8]
**Describe the bug**
JQ Filter relies on fields that are not provided by the LR Webhook Beat.
Thanks to @Jt3kt for catching this one!

**To Reproduce**
1. Configure a stream to be collected via Webhook Beat
2. Run tail and do mapping of fields
3. Deploy Stream
4. Send data to Webhook beat via HTTP

**Expected behavior**
The logs should be caught by the JQ filter, then be transformed and sent to the SIEM.

**Screenshots**
Log sample sent:
``` json
{"SAMPLE":{"LOG":"BLA BLA"}}
```

How the Webhook Beat sends it to the Open Collector:
``` json
{
  "@timestamp":"2022-03-28T16:34:43.545Z",
  "@metadata":{
    "beat":"webhookbeat",
    "type":"doc",
    "version":"6.6.0"
  },
  "fullyqualifiedbeatname":"webhookbeat_254_EH_254a3",
  "beat":{
    "version":"6.6.0",
    "name":"66b638a55393",
    "hostname":"66b638a55393"
  },
  "host":{
    "name":"66b638a55393"
  },
  "message":"{\"SAMPLE\":{\"LOG\":\"BLA BLA\"}}"
}
```

How the SIEM receives it:
```
2022-03-28T17:06:29.949Z f00387463eb7 ./image.binary[1]: |beatname=webhookbeat|device_type=webhookbeat|fullyqualifiedbeatname=webhookbeat_254_EH_254a3|original_message={"@metadata":{"beat":"webhookbeat","type":"doc","version":"6.6.0"},"@timestamp":"2022-03-28T17:06:28.935Z","beat":{"hostname":"3a0c080a51d8","name":"3a0c080a51d8","version":"6.6.0"},"fullyqualifiedbeatname":"webhookbeat_254_EH_254a3","host":{"name":"3a0c080a51d8"},"message":"{\"SAMPLE\":{\"LOG\":\"BLA BLA\"}}"}|
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 99.0.4844.82 (Official Build) (64-bit)

**Additional context**

**Comment by TonyMasse** [comment]
![image](https://user-images.githubusercontent.com/50835088/160451513-f24491ab-5768-4b58-b47d-ad0df5c0e0b7.png)

**Comment by TonyMasse** [comment]
Fixed by commit 090a26d .
Closing.
